meterfs/meterfs.c

```diff
@@ -768,6 +768,10 @@ int meterfs_devctl(meterfs_i_devctl_t *i, meterfs_o_devctl_t *o, meterfs_ctx_t *
 		case meterfs_chiperase:
 			meterfs_powerctrl(1, ctx);
 			ctx->partitionErase();
+			ctx->filecnt = 0;
+			ctx->h1Addr = 0;
+			ctx->hcurrAddr = 0;
+			ctx->offset = 0;
```

Comment:
We shouldn't change offset here. Offset is an information about partition placement (there can be more than one meterfs on a single flash).